frontend/src/App.js

```diff
@@ -52,32 +60,41 @@ useEffect(() => {
     setTime({ms:0, s:0, m:0})
   };
 
-  // var jest zly, naprawione
-  let updMs = time.ms, updS = time.s, updM = time.m;
+  let updateMiliseconds = time.ms, updateSeconds = time.s, updateMinutes = time.m;
 
   const run = () => {
-    if(updS === 60){
-      updM ++;
-      updS = 0;  
+    if(updateSeconds === 60){
+      updateMinutes ++;
+      updateSeconds = 0;  
```

Comment:
updateMinutes wskazuje itp wskazują jakby to była funkcja a nie zmienna, raczej nazwałbym
let currentMiliseconds etc.

Reply:
ok